src/app/(profile)/_actions/countProfileView.ts

```diff
@@ -0,0 +1,18 @@
+'use server'
+import {
+  getProfileById,
+  incrementProfileViewCountById,
+} from '@/backend/profile/profile.service'
+import { withSentry } from '@/utils/errHandling'
+
+export const countProfileView = withSentry(
+  async (profileId: string, userProfileId?: string) => {
```

Comment:
it is not `userProfileId` - let's change it to `userId` because you pass `userId` not `profileId`